### PR TITLE
Update scrolls.xml

### PR DESCRIPTION
**Pull request by DariaElagina**
see [https://github.com/BetaMasaheft/Documentation/issues/1379#event-4313735152](url)

### CODE REVIEW

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: pages/scrolls.xml [hunk 1]
```diff
@@ -16,17 +16,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+        <revisionDesc>
+            <change who="PL" when="2020-06-22">Created page</change>
+            <change who="DE" when="2021-02-10">Added the examples on encoding of physical description.</change>    
         </revisionDesc>
     </teiHeader>
     <text>
         <body xml:id="scrolls">
             <div>
+              <head>Description of Scrolls</head>
                 <p>We treat Scrolls in the same way as we do with <ref target="manuscripts">Manuscripts</ref> and any other written artefacts,
```
Comment (thea-m) [inline pages/scrolls.xml:29]: I know it is an old sentence, but let's take the opportunity to make it correct:
"We treat scrolls in the same way as `<ref target="manuscripts">codices</ref>`"

File: pages/scrolls.xml [hunk 2]
```diff
@@ -16,17 +16,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                    and only add in <gi>objectDesc</gi> the value Scroll for <att>form</att>.</p>
-               <p>when structuring the text transcription of your scroll 
+                    and only add in <gi>objectDesc</gi> the value Scroll for <att>form</att> However,
```
Comment (thea-m) [inline pages/scrolls.xml:30]: I think . after `</att>`  is missing

File: pages/scrolls.xml [hunk 3]
```diff
@@ -16,17 +16,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                              <width>120</width>
+                           </dimensions>
+                           <dimensions type="outer" unit="mm" xml:id="strip3">
+                              <height>540</height>
+                              <width>97</width>
+                           </dimensions>
+             </egXML>
+                    </p>
+                    <p>The assigned <gi>xml:id</gi>s can be referenced to while encoding
+                  of other features. For example:</p>
```
Comment (thea-m) [inline pages/scrolls.xml:57]: no of?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you :)

**Review by eu-genia — APPROVED** [review]

**Pull request merged**